apps/organisations/views.py

```diff
@@ -283,13 +289,11 @@ def generate_image(self, data):
 
         # add text using width to center
         image.text(((sharepic_format['img_min_width'] - title_width) / 2,
-                    sharepic_format['title_y']),
-                   data['title'], fill=(0, 0, 0), font=font)
+                    sharepic_format['title_y']), title, fill=(0, 0, 0),
+                   font=font)
```

Comment:
The brackets are a bit hard to read. Can we use the same format for this one and the one below? I personally like:
```
some_function(
    stuff_here,
    (stuff in more braces, bla),
    and_so_on
)
```
That will not quite work here because of the longer brace in the brace, but at least have everything eles in single lines?